README.md

```diff
@@ -30,3 +31,18 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+## Runtime Solution
+With three for loops, two running through $n*n$ and one only running through one $n$, we have $n^5$, along with three mystery calls with an arguement of $n / 3$. 
+So the recurrence relationship is $1$ if $n <= 1$ and $mystery(n-3) + n^5$ if $n>1$ and we can start solving. 
+
+$T(n/3) = 3T(n/3/3) + (n/3)^5 \Rightarrow 3T(n/9) + n^5/243$ and then we can insert that into $3T(n/3) + n^5$
+$3(3T(n/9) + n^5/243) + n^5 \Rightarrow 9T(n/9) + (1 + 3(1/243))n^5$ But we can let $(1 + 3(1/243))$ be a constant called $c$
+$9T(n/3) + cn^5 = 9T(n/3/9) + (n/3)^5 \Rightarrow 9T(n/27) + cn^5/243$ and then we can insert that into $3T(n/3) + n^5$
+
+$3T(9T(n/27) + cn^5/243) + n^5 \Rightarrow 27T(n/27) + (1 + 3c(1/243))n^5$ and again we can all that mess of numbers in front of $n^5$ $c$ and we have a pattern! 
```

Comment:
Is the $c$ really a constant, or are you adding a new term for every recursive expansion?

Reply:
Right, it wasn't as simple as that, thanks!